packages/tradershub/src/flows/FirstSignup/FirstSignupWrapper/FirstSignupWrapper.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
@sergei-deriv please rename the file and folder name
packages/tradershub/src/flows/Signup/SignupWrapper/SignupWrapper.tsx